app/Http/Controllers/Knowledge/DestroyController.php

```
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\Knowledge;
+
+use App\Models\Knowledge;
+use Illuminate\Http\Request;
+
+class DestroyController
+{
+    public function __invoke(Request $request, Knowledge $knowledge)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)